src/java.desktop/share/classes/java/awt/MenuComponent.java

```diff
@@ -449,8 +428,6 @@ private void readObject(ObjectInputStream s)
     {
```

Comment:
`@SuppressWarnings("removal")` can be removed

Reply:
ok